images/syncer/Dockerfile

```diff
@@ -48,6 +48,9 @@ RUN go build -a -ldflags="-w -s -extldflags=static -X sigs.k8s.io/vsphere-csi-dr
 ################################################################################
 FROM ${BASE_IMAGE}
 
+RUN tdnf -y remove toybox
+RUN tdnf -y upgrade
```

Comment:
Any reason why we dont run upgrade command in the CSI Dockerfile?

Reply:
CSI driver is using a base image that already contains upgraded packages - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/images/driver-base/Dockerfile#L18-L19 

In the past, we missed adding this for the syncer container. syncer container is using photon:3.0 as the base image. - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/images/syncer/Dockerfile#L20

Reply:
syncer does not need all packages we need for driver image which is used by node daemonset as well which includes nfsutil.

nfsutil adds a lot of other packages.

```
RUN tdnf -y install nfs-utils
 ---> Running in 304e87df4cca
Refreshing metadata for: 'VMware Photon Extras 3.0 (x86_64)'
Refreshing metadata for: 'VMware Photon Linux 3.0 (x86_64)'
Refreshing metadata for: 'VMware Photon Linux 3.0 (x86_64) Updates'
Installing:
pcre-libs                x86_64       8.44-1.ph3       photon-updates 279.61k 286320
libgcrypt                x86_64       1.8.5-1.ph3      photon-updates   1.18M 1232716
glib                     x86_64       2.58.0-4.ph3     photon-updates   3.11M 3265152
libstdc++                x86_64       7.3.0-4.ph3      photon         1.62M 1694772
libgpg-error             x86_64       1.32-1.ph3       photon       203.31k 208186
kmod                     x86_64       25-1.ph3         photon       247.49k 253427
elfutils                 x86_64       0.176-1.ph3      photon-updates   1.74M 1821041
systemd                  x86_64       239-29.ph3       photon-updates  20.20M 21182004
libselinux               x86_64       2.8-2.ph3        photon-updates 166.24k 170226
util-linux-libs          x86_64       2.32.1-3.ph3     photon-updates 723.08k 740432
cracklib                 x86_64       2.9.6-8.ph3      photon        43.37k 44408
Linux-PAM                x86_64       1.3.0-1.ph3      photon         1.06M 1113660
shadow                   x86_64       4.6-5.ph3        photon-updates   1.82M 1904777
libffi                   x86_64       3.2.1-6.ph3      photon        39.28k 40224
python3-libs             x86_64       3.7.5-7.ph3      photon-updates  22.96M 24079495
xz                       x86_64       5.2.4-1.ph3      photon       166.45k 170445
ncurses                  x86_64       6.1-2.ph3        photon-updates 357.89k 366475
python3                  x86_64       3.7.5-7.ph3      photon-updates   2.94M 3084254
libsepol                 x86_64       2.8-1.ph3        photon       696.09k 712800
device-mapper-libs       x86_64       2.02.187-1.ph3   photon-updates 357.14k 365712
rpcbind                  x86_64       1.2.5-1.ph3      photon        92.85k 95077
pkg-config               x86_64       0.29.2-2.ph3     photon       610.51k 625164
libtirpc                 x86_64       1.1.4-1.ph3      photon       208.85k 213863
libevent                 x86_64       2.1.8-1.ph3      photon       754.73k 772840
keyutils                 x86_64       1.5.10-1.ph3     photon       140.36k 143726
nfs-utils                x86_64       2.3.3-3.ph3      photon-updates   1.25M 1311178
Total installed size:  62.85M 65898374
```